guides/doc-Hammer_CLI/topics/con_introduction-to-hammer.adoc

```diff
@@ -0,0 +1,457 @@
+----
+
+[[sect-CLI_Guide-Setting_a_Default_Organization]]
+== Setting a default organization and location
+
+Many `hammer` commands are organization specific.
+You can set a default organization and location for `hammer` commands so that you do not have to specify them every time with the `--organization` and `--location` options.
+
+Specifying a default organization is useful when you mostly manage a single organization, as it makes your commands shorter.
+However, when you switch to a different organization, you must use `hammer` with the `--organization` option to specify it.
+
+.Procedure
+. Set a default organization:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# hammer defaults add --param-name organization \
+--param-value _"Your_Organization"_
+----
++
+You can find the name of your organization with the `hammer organization list` command.
+. Optional: Set a default location:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# hammer defaults add --param-name location \
+--param-value _"Your_Location"_
+----
++
+You can find the name of your location with the `hammer location list` command.
+
+.Verification
+. Review the currently specified default settings:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+# hammer defaults list
+----
+
+[[sect-CLI_Guide-Configuring_Hammer]]
+== Configuring Hammer
+
+The default location for global `hammer` configuration is:
+
+* */etc/hammer/cli_config.yml* for general `hammer` settings
+
+* */etc/hammer/cli.modules.d/* for CLI module configuration files
+
+You can set user specific directives for `hammer` (in *~/.hammer/cli_config.yml*) as well as for CLI modules (in respective *.yml* files under *~/.hammer/cli.modules.d/*).
+
+To see the order in which configuration files are loaded, as well as versions of loaded modules, use:
+
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+$ hammer -d --version
+----
+
+[NOTE]
+====
+Loading configuration for many CLI modules can slow down the execution of `hammer` commands.
+In such a case, consider disabling CLI modules that are not regularly used.
+====
+Apart from saving credentials as described in xref:sect-CLI_Guide-Authentication[], you can set several other options in the *~/.hammer/* configuration directory.
+For example, you can change the default log level and set log rotation with the following directives in *~/.hammer/cli_config.yml*.
+These directives affect only the current user and are not applied globally.
+
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+:log_level: 'warning'
+:log_size: 5 #in MB
+----
+
+Similarly, you can configure user interface settings.
+For example, set the number of entries displayed per request in the Hammer output by changing the following line:
+
+----
+:per_page: 30
+----
+
+This setting is an equivalent of the `--per-page` Hammer option.
+
+[[sect-CLI_Guide-Configuring_Hammer_Logging]]
+== Configuring Hammer logging
```

Comment:
This is specifically for [hammer-cli-foreman-admin](https://github.com/theforeman/hammer-cli-foreman-admin) which is a very unusual plugin. Normally Hammer is a tool that talks to the Foreman API over the network. hammer-cli-foreman-admin actually operates on the server itself. I'd love to get rid of it.

We already have a section that can change the log levels (https://docs.theforeman.org/nightly/Administering_Project/index-katello.html#Logging_and_Reporting_Problems_admin). I'd be my preference to use that instead as the single way to configure logging.